### ISSUE TITLE
XBuildResolver fails on OSX with Mono 4.4.0

### ISSUE THREAD

**Issue by eatdrinksleepcode** [Ullink/gradle-msbuild-plugin#62]
On my OSX system (El Capitan), I have the following framework versions in my Mono 4.4.0 installation:

```
/Library/Frameworks/Mono.framework/Versions/4.4.0/lib/mono:
2.0-api
3.5-api
4.0
4.0-api
4.5
4.5-api
```
Note the "-api" suffix on several of the versions. I am not entirely sure what they are for, but they seem to be normal for Mono 4.4.0. XBuildResolver fails to handle these suffixes correctly and throws an exception. The problem is that XBuildResolver initially uses the undocumented Groovy class VersionNumber to filter the list of directories under /lib/mono; it then assumes that it can parse any of the matching directory names as Floats. However VersionNumber allows a "-qualifier" to be appended to the version, which matches the "-api" in the directories above; but the "-api" cannot be parsed as a Float, causing the exception.

**Comment by eatdrinksleepcode** [comment]
Is there any chance of getting a build with this fix pushed to Maven Central? I would like to be able to build IL Repack on OSX, for which this fix is required (gluck/il-repack#156).

**Comment by gluck** [comment]
The 2.15 release has been done for some time (on bintray/gradle plugin repository), but the sync of maven central was missed (apparently), this is now fixed and sync'ed to maven central, though bintray is the recommended source of gradle plugins :)

Thx for noticing !
